barretenberg/cpp/src/barretenberg/vm2/common/testing/fixtures.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <vector>
+
+#include "barretenberg/vm2/generated/flavor_settings.hpp"
+
+namespace bb::avm2::constraining {
+
+using FF = AvmFlavorSettings::FF;
+inline std::vector<FF> random_fields(size_t n)
```

Comment:
Since these are not templated they should have their own cpp, but it's ok to merge if you get a green CI, we can do it later.